### DISCUSSION TITLE
Welcome to popschools.org Discussions!

### DISCUSSION THREAD

**Discussion by prophen** [PopSchools/popschools.org#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tuganimana** [comment]
Thank you @prophen kema !!

**Reply by prophen** [reply]
Thank you for sticking around, @tuganimana !

**Comment by prophen** [comment]
Okay, get started with an intro...

I'm Nikema. I founded PopSchools in 2018 with a completely different focus. I wanted to create family-friendly, tech-enabled, co-working and co-learning spaces. Basically, WeWork + [unschooling](https://www.naturalchild.org/articles/guest/earl_stevens.html).

Once I started getting more involved in the software development world, I encountered some organizations and cultural norms that I found disturbing and exploitative. I pivoted and began to focus on creating safe and equitable spaces for people pursuing a tech career from "non-traditional" paths.

I want to do as much as I can with PopSchools to steer people away from the pitfalls and hazards I've encountered. I want to see people level up and have satisfying careers without wasting time and future earnings in low-quality coding schools. Voices like mine are underrepresented and need to be heard by people who are coming from a place similar to mine. 

I'm a full-time single mom, I have no college degrees in CS or otherwise. I'm Black, a woman, and neurodivergent (ADHD). The usual advice generally doesn't apply to my situation. I'm here to give a voice to that experience.

I want to work with you. I want to grow this community and grow with this community.
